### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by cinfwatd** [leftstick/teams-ubuntu#12]
Hi @leftstick nice app I must say. I have it running but don't have any popup notification for messages. Do we have that already or am I missing something?

**Comment by leftstick** [comment]
Maybe [desktop notification](https://github.com/leftstick/teams-ubuntu/issues/5)?

**Comment by cinfwatd** [comment]
Yes [desktop notifications](https://github.com/leftstick/teams-ubuntu/issues/5) @leftstick.

**Comment by leftstick** [comment]
@cinfwatd  so, does it solve your issue by opening `banner`?

**Comment by cinfwatd** [comment]
@leftstick No it doesn't solve my issue. I enabled banner in `notifications` but still don't get Desktop Notifications.

**Comment by filipows** [comment]
Same here, desktop notifications don't work for me. Has anyone found a way around it?

**Comment by mitulvc** [comment]
Hi @leftstick awesome app, Is support desktop notification?
Has anyone found the solution?

**Comment by ogghi** [comment]
Any news on this? Just missed some conversation because of no notification, not even the task bar symbol...
